datasetinsights/constants.py

```diff
@@ -53,3 +53,6 @@
     "allow_extra_args": True,
 }
 DEFAULT_DATASET_VERSION = "latest"
+
+MLFLOW_TRACKER = "mlflow"
```

Comment:
These variables are better stored in `io/tracker/factory.py` as you will have `KUBEFLOW_TRACKER`, `TENSORBOARD_TRACKER` in the future.